src/pages/Home/Home.jsx

```diff
@@ -1,13 +1,12 @@
-import { Link } from 'react-router-dom';
-import { HelloWorld, Counter } from '../../components';
-import { routes } from '../../routes/Routes';
+import { ReactComponent as Logo } from '../../Images/logo.svg';
+import { Button } from '../../components/index';
+import './Home.scss';
 
 export const HomePage = () => {
```

Comment:
Add TODO Comment that we actually need to connect this button with google log in